### PR TITLE
feat(storage-offers): handles offers actions with new confirmations feat

### PR DESCRIPTION
**Pull request by farinalvaro**
Uses new confirmations mechanism for offer creation, edition and cancelation.
Related to #607

### CODE REVIEW

**Review by julianmrodri — APPROVED** [review]
LGTM

File: src/components/organisms/storage/sell/OfferCreation.tsx [hunk 1]
```diff
@@ -109,44 +101,33 @@ const OfferCreation: FC = () => {
...
-          id: 'contract',
-        } as LoadingPayload,
-      })
+      setIsTxInProgress(false)
     }
   }
 
   const onProcessingComplete = (): void => {
-    setIsProcessing(false)
+    history.push(ROUTES.STORAGE.MYOFFERS.BASE)
```
Comment (jurajpiar) [inline src/components/organisms/storage/sell/OfferCreation.tsx:128]: redirect should prob by done in OfferEditContext

File: src/components/pages/storage/myoffers/StorageEditOfferPage.tsx [hunk 2]
```diff
@@ -80,42 +90,30 @@ const StorageEditOfferPage: FC<{}> = () => {
...
-          isLoading: false,
-          id: 'contract',
-        } as LoadingPayload,
-      } as any)
     }
   }
 
   const onProcessingComplete = (): void => {
-    setIsProcessing(false)
+    history.push(ROUTES.STORAGE.MYOFFERS.BASE)
```
Comment (jurajpiar) [inline src/components/pages/storage/myoffers/StorageEditOfferPage.tsx:114]: same

File: src/components/pages/storage/myoffers/StorageMyOffersPage.tsx [hunk 3]
```diff
@@ -132,11 +126,16 @@ const StorageMyOffersPage: FC = () => {
       type: 'SET_OFFER',
       payload: offer as SetOfferPayload,
     })
-    history.push(ROUTES.STORAGE.MYOFFERS.EDIT.BASE)
+    history.push(ROUTES.STORAGE.MYOFFERS.EDIT)
```
Comment (jurajpiar) [inline src/components/pages/storage/myoffers/StorageMyOffersPage.tsx:129]: same

**Review by jurajpiar — APPROVED** [review]

**Comment by farinalvaro** [comment]
Converting to draft so I can add the new useConfirmations hook added on https://github.com/rsksmart/rif-marketplace-ui/pull/625 before merging

File: src/components/organisms/storage/sell/OfferCreation.tsx (hunk 1 above)
Comment (farinalvaro) [reply src/components/organisms/storage/sell/OfferCreation.tsx:128]: Hmm why? I find it a bit clearer to do it in the component that triggers the action, but I might be missing something

**Review by jurajpiar — APPROVED** [review]

**Review by julianmrodri — APPROVED** [review]
LGTM

**Pull request merged**